modules/standard/IO.chpl

```diff
@@ -5196,7 +5207,8 @@ proc _channel.readbytes(ref bytes_out:bytes, len:int(64) = -1):bool throws {
   :returns: a new ``bytes`` containing up to the next ``maxSize`` bytes
               from the ``fileReader``
 
-  :throws SystemError: Thrown if a ``bytes`` could not be read from the ``fileReader``.
+  :throws SystemError: Thrown if a ``bytes`` could not be read from the ``fileReader``
```

Comment:
Does this need a `throws UnexpectedEofError: ...` as well?

Reply:
It cannot throw a `UnexpectedEofError` because it only throws an error created from `createSystemError` which can only create `SystemError`s. I suppose that means I could likely remove the `:throws UnexpectedEofError` from `readbytes` above as well, since even though it uses `_ch_ioerror` which can create an `UnexpectedEofError`, it isn't doing anything different from this version. I will do that.